core/operation/src/main/java/uk/gov/gchq/gaffer/operation/data/EntitySeed.java

```diff
@@ -49,23 +51,27 @@ public boolean equals(final Object o) {
         if (this == o) {
```

Comment:
The reference name here should be longer than one character - could you update this to:

```
public boolean equals(final Object obj) {
    // ...
}
``` 

Could you also update the other ```equals()``` methods too?